bitnami/keycloak/values.yaml

```
@@ -655,3 +655,78 @@ externalDatabase:
   ## Database name
   ##
   database: bitnami_keycloak
+
+##
+## Below is the implementation of Keycloak Configuration as Code.
+##
+KcCasC:
+  ## Whether to enable Keycloak Configuration as Code.
+  ##
+  enabled: false
+
+  image:
+    repository: docker.io/adorsys/keycloak-config-cli
+    tag: "v{{ .Chart.AppVersion }}-11.0.3"
```

We have released the bitnami/keycloak-config-cli container image, please update the values and test that image on your side. Let us know about the results.

```suggestion
  ## Bitnami keycloak-config-cli image version
  ## ref: https://hub.docker.com/r/bitnami/keycloak-config-cli/tags/
  ##
  image:
    registry: docker.io
    repository: bitnami/keycloak-config-cli
    tag: 3.1.0-debian-10-r0
```